README.md

```diff
@@ -12,14 +12,15 @@ BigQuery.
 This repository is broken up into:
 
 *   [Dashboards](/dashboards) - Pre-built dashboards for common use cases
+*   [Notebooks](/notebooks) - Colab notebook tutorials.
```

Comment:
Remove line 15, and maybe add "and notebooks" to line 14

Reply:
Is that appropriate?

My concern is that this notebook is a _tutorial_ for ad-hoc analysis.  This is not a dashboard.

Reply:
Updated